modules/local/report.nf

```diff
@@ -804,24 +804,57 @@ def table_values(file_path, header_p, seperator, headers=null){
     def converted_data = [:]
     def idx = 0
     def lines_read = false
+    def missing_value = "NoData"
+    def default_index_col = "__default_index__"
     file_path.withReader{
         String line
         if(header_p){
             header = it.readLine()
-            split_header = header.tokenize(seperator)
+            split_header = header.split(seperator)
+
+            def missing_headers = 0
+            if(split_header.size() > 1){
+                for(col_header in split_header[1..-1]){ // skip first column as it is allowed to be empty
+                    if(!col_header){
+                        missing_headers++;
+                    }
+                }
+            }
+
+            if(missing_headers){
```

Comment:
Instead of relying on 0 being false, non-zero being true, could you expliclity state what you are checking for? In this case:

```groovy
if (missing_headers > 0)
```

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/83/commits/049c43c917af70478960c9ec3f1a83b49b931cf9